### PR TITLE
ET/FSE: Remove the "Showcase New Blocks" module

### LINKED ISSUE #46530: Editing Toolkit: Remove "New" category
The "New" category in the block inserter tends to push down valuable blocks and adds noise for most people. @mtias suggested removing it.

Introduced in #41822.
See pbAPfg-Xn-p2

### PR DESCRIPTION
**Pull request by mreishus**
fixes #46530

#### Changes proposed in this Pull Request

* Remove the "Showcase New Blocks" module
Added in #41822
Requested for removal in #46530 ()

#### Testing instructions

* Have sandbox set up and refreshed
* `cd apps/editing-toolkit`
* `yarn run dev --sync`
* Visit sandbox and edit post
* Look in left side inserter for "New" section - should be gone

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js [hunk 1]
```diff
@@ -72,9 +72,10 @@ const copyStylesToIframe = ( srcDocument, targetiFrameDocument ) => {
  * @param {object} props.className CSS class to apply to component
  * @param {string} props.bodyClassName CSS class to apply to the iframe's `<body>` tag
  * @param {number} props.viewportWidth pixel width of the viewable size of the preview
- * @param {Array}  props.blocks array of Gutenberg Block objects
+ * @param {Array} props.blocks array of Gutenberg Block objects
  * @param {object} props.settings block Editor settings object
  * @param {Function} props.setTimeout safe version of window.setTimeout via `withSafeTimeout`
+ * @param props.title Template Title - see #39831 for details.
```
Comment (mreishus) [inline apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js:78]: `yarn run dev` in FSE automatically added this param, I needed to add a description to get it to pass the commit hook.

Comment (obenland) [reply apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js:78]: Does that need a `{string}` prepended?

**Comment by getdave** [comment]
I tried to test this but the `yarn run dev --sync` didn't work. Failed with

> rsync error: error in rsync protocol data stream (code 12) at /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-54.120.1/rsync/io.c(453) [sender=2.6.9]

**Comment by mreishus** [comment]
> I tried to test this but the `yarn run dev --sync` didn't work. Failed with
> 
> > rsync error: error in rsync protocol data stream (code 12) at /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-54.120.1/rsync/io.c(453) [sender=2.6.9]

@getdave: Does `ssh wpcom-sandbox` work?  If no, you need to add an entry to your `~/.ssh/config` to make it work.

```
Host wpcom-sandbox
  User usernamegoeshere
  HostName host.name.goes.here
```

If it does work, it might be something else like rsync version (try using brew to install a newer version)?  Another alternative is to use `yarn run dev` to create the files, and then manually find your own way to copy the files in (local) `./wp-calypso/apps/editing-toolkit/editing-toolkit-plugin/` to (sandbox) `/home/wpcom/public_html/wp-content/plugins/full-site-editing-plugin/dev`.

File: apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js (hunk 1 above)
Comment (mreishus) [reply apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js]: Fixed in 5e8ef98e6

**Comment by cpapazoglou** [comment]
> I tried to test this but the `yarn run dev --sync` didn't work. Failed with
> 
> > rsync error: error in rsync protocol data stream (code 12) at /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-54.120.1/rsync/io.c(453) [sender=2.6.9]

try running `composer install` also

**Review by cpapazoglou — APPROVED** [review]
LGTM!

**Pull request merged**